openownership/data-standard
# Schema tests with invalid data

# Overview

- What does this pull request do?

Completes the test structure so that the schema constraints can be tested against invalid data files.

Files are mapped to the expected errors in a csv file, for ease of use when making schema changes.

- How can a reviewer test or examine your changes?

Run the tests, mess about with the invalid-statements files to see that it works.

- Who is best placed to review it?

Closes #572 

## Documentation & Release

Update to bods-dev-handbook imminent

- [n/a ] I've checked that the documentation builds without failing. See: https://github.com/openownership/data-standard#building-the-documentation-locally
- [n/a] I've thought about how and when this needs to be released. See:
      https://openownership.github.io/bods-dev-handbook/standard_releases.html      
- [n/a] I've updated the changelog, if necessary.
- [n/a] I've updated [reference.rst](https://standard.openownership.org/en/latest/schema/reference.html) to reflect any changes to schema structure or naming.
